index.html

```diff
@@ -37,9 +45,8 @@ <h2>Connect</h2>
             </ul>
         </section>
         
-
+        <footer></footer>
```

Comment:
hey, did you use footer here? 
which of the following  <footer> element did you contain?

authorship information
copyright information
contact information
sitemap
back to top links
related documents
afaik, we can have several <footer> elements in one document.